modules/openthread/platform/dhcp6_pd.c

```diff
@@ -63,9 +63,9 @@ void otPlatInfraIfDhcp6PdClientSend(otInstance *aInstance,
 	VerifyOrExit(dhcpv6_pd_client_sock != -1 && aInfraIfIndex == ail_iface_idx);
 	uint16_t length = otMessageGetLength(aMessage);
 	struct net_sockaddr_in6 dest_sock_addr = {.sin6_family = NET_AF_INET6,
-					      .sin6_port = net_htons(DHCPV6_SERVER_PORT),
-					      .sin6_addr = NET_IN6ADDR_ANY_INIT,
-					      .sin6_scope_id = 0};
+						  .sin6_port = net_htons(DHCPV6_SERVER_PORT),
+						  .sin6_addr = NET_IN6ADDR_ANY_INIT,
+						  .sin6_scope_id = 0};
```

Comment:
Nitpicking here, if you'd add a trailing comma to the last item, it would be formatted like:

```c
	struct net_sockaddr_in6 dest_sock_addr = {
		.sin6_family = NET_AF_INET6,
		.sin6_port = net_htons(DHCPV6_SERVER_PORT),
		.sin6_addr = NET_IN6ADDR_ANY_INIT,
		.sin6_scope_id = 0,
	};
```

Which is a best nicer IMO.